tests/population/test_mortality.py

```diff
@@ -33,6 +51,169 @@ def test_mortality_default_lookup_configuration(
+
+    model = DiseaseModel("test", initial_state=healthy, states=[healthy, mortality_state])
+    sim = InteractiveContext(
+        components=[bp, mortality, model], plugin_configuration=base_plugins, setup=False
+    )
+    override_config = {
+        "population": {
+            "population_size": start_population_size,
+            "include_sex": "Male",
+        },
+        "mortality": {
+            "all_cause_mortality_rate": {
+                "value": 0.8,
+            }
+        },
+    }
+    sim.configuration.update(override_config)
+    sim.setup()
+    sim.step()
+    # Only other causes and sick for cause of death
+    pop1 = sim.get_population()
+    for cause_of_death in ["other_causes", "sick"]:
+        dead = pop1.loc[pop1["cause_of_death"] == cause_of_death]
+        # Disease model seems to set mortality rate for that diesease back to 0
+        # if a simulant dies from it
+        rates = mortality.mortality_rate(pop1.index)[cause_of_death].unique()
+        for mortality_rate in rates:
+            if mortality_rate == 0:
+                continue
+            else:
+                rate = mortality_rate
+            if cause_of_death == "sick":
+                rate *= 0.5  # prevalence
+            fuzzy_checker.fuzzy_assert_proportion(
+                name=f"test_mortality_rate_{cause_of_death}",
+                observed_numerator=len(dead),
+                observed_denominator=len(pop1),
+                target_proportion=rate,
+            )
+
+
+def test_mortality_ylls(setup_sim_with_pop_and_mortality):
+    sim, bp, mortality = setup_sim_with_pop_and_mortality
+    sim.step()
+    pop1 = sim.get_population()
+
+    dead_idx = pop1.index[pop1["alive"] == "dead"]
+    ylls = pop1.loc[dead_idx, "years_of_life_lost"]
+    assert (ylls == mortality.lookup_tables["life_expectancy"](dead_idx)).all()
+
+
+def test_no_unmodeled_causes(setup_sim_with_pop_and_mortality):
+    sim, bp, mortality = setup_sim_with_pop_and_mortality
+    # No unmodeled causes by default
+    assert isinstance(
+        mortality.lookup_tables["unmodeled_cause_specific_mortality_rate"], ScalarTable
+    )
+    assert mortality.lookup_tables["unmodeled_cause_specific_mortality_rate"].data == 0.0
+
+
+def test_unmodeled_causes(full_simulants, base_plugins, generate_population_mock):
+    start_population_size = len(full_simulants)
+    generate_population_mock.return_value = full_simulants.drop(columns=["tracked"])
+    bp = BasePopulation()
+    mortality = Mortality()
+
+    sim = InteractiveContext(
+        components=[bp, mortality], plugin_configuration=base_plugins, setup=False
+    )
+    override_config = {
+        "population": {
+            "population_size": start_population_size,
+            "include_sex": "Male",
+        },
+        "mortality": {
+            "unmodeled_cause_specific_mortality_rate": {
+                **{"unmodeled_causes": ["low_birth_weight"]},  # , "malnutrition", "malaria"],
+            }
+        },
```

Comment:
Shouldn't this config look like this:
```suggestion
        "mortality": {
            "unmodeled_causes": ["low_birth_weight"]},  # , "malnutrition", "malaria"],
        },
```

Also, why are malnutrition and malaria commented out?

Reply:
Good catch.